File: launchdarkly/php/src/GetAIConfigMetricsExample.php
<?php

namespace OSEG\LaunchDarklyExamples;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use LaunchDarkly;

$config = LaunchDarkly\Client\Configuration::getDefaultConfiguration();
$config->setApiKey("ApiKey", "YOUR_API_KEY");

try {
    $response = (new LaunchDarkly\Client\Api\AIConfigsBetaApi(config: $config))->getAIConfigMetrics(
        ld_api_version: null,
        project_key: null,
        config_key: null,
        from: 123,
        to: 456,
        env: null,
    );

    print_r($response);
} catch (LaunchDarkly\Client\ApiException $e) {
    echo "Exception when calling AIConfigsBeta#getAIConfigMetrics: {$e->getMessage()}";
}

File: launchdarkly/ruby/src/GetAIConfigMetricsExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["ApiKey"] = "YOUR_API_KEY"
end

begin
    response = LaunchDarklyClient::AIConfigsBetaApi.new.get_ai_config_metrics(
        nil, # ld_api_version
        nil, # project_key
        nil, # config_key
        123, # from
        456, # to
        nil, # env
    )

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling AIConfigsBetaApi#get_ai_config_metrics: #{e}"
end
